projects/hslayers/shared/add-data/url/hs.source.WfsSource.ts

```diff
@@ -74,27 +78,30 @@ export class WfsSource extends VectorSource {
           hsUtilsService.paramsToURLWoEncode(params),
         ].join('?');
         url = hsUtilsService.proxify(url);
-        this.dispatchEvent('featuresloadstart');
-        const response = await lastValueFrom(
-          http.get(url, {responseType: 'text'}),
-        );
-        if (response) {
-          const features = readFeatures(
-            response,
-            map_projection,
-            data_version,
-            responseFeatureCRS,
-          );
-          (this as VectorSource).addFeatures(features);
-          this.dispatchEvent(
-            new ObjectEvent('propertychange', 'loaded', false),
-          );
-          this.dispatchEvent('featuresloadend');
-        } else {
-          this.dispatchEvent('featuresloaderror');
-        }
+        //this.dispatchEvent('featuresloadstart');
+
+        http.get(url, {responseType: 'text'}).subscribe({
+          next: (response) => {
+            const features = readFeatures(
+              response,
+              map_projection,
+              data_version,
+              responseFeatureCRS,
+            );
+            (this as VectorSource).addFeatures(features);
+            this.dispatchEvent(
+              new ObjectEvent('propertychange', 'loaded', false),
```

Comment:
I keep thinking about the 'loaded' property purpose. I found it is used only in `WfsSource` and `SparqlJson` sources. In `WfsSource`, it is never set to `true`, or if we change the line as per above comment, then it will set to `true` after each request to WFS server finishes. In `SparqlJson`, it is set to `true` only after all concurrent requests finish. I would like to keep the behaviour consistent. Which approach makes better sense to you? `layerLoads` and `layerLoadings` subjects (and their frequency of casting) are bound to it.